src/docs/source/docs/memo.md

```diff
@@ -0,0 +1,202 @@
+Memo
+=====================
+
+Steps to Create an Memo
+---------------------------------------
+1. The process of adding any variety of content to PECE begins by logging in with your screen name and password and clicking “Create Content” in the top right corner.
+
+    ![](media/memo_1.png)
+
+2. You should now see the "Add content" page.
+
+    ![](media/memo_2.png)
+
+    As you scroll down the page, you can find thumbnails 
+    for creating different forms of new content. Each is organised under a category. "Memo" can be found under the category "Analyze". "Memo" is one of the type of content. A memo is a first draft of a piece of writing, meant to synthesize ideas generated throughout data analysis. Upon publishing a memo, that memo is opened for comments from the platform's community.
+
+    ![](media/memo_3.png)
+
+3. To create a memo, click on the "Memo" thumbnail. 
+
+    ![](media/memo_4.png)
+
+4. This takes you to the "Create Memo" page. You should now have access to all of the fields you need to fill out in order to create a new memo. First, type the title of your memo in the “Title” field.
+
+5. Type your memo into the "Body" field.
+    To provide a summary of your memo, click the "Edit summary" link next to "Body".
+
+6. Next are two tabs: "Permissions, Authorship, and Groups" and "Contribution Metadata". The "Permissions, Authorship, and Groups" should be automatically opened. 
+
+    In the "Edit Permissions and Authorship" section, you should be automatically added as a contributor as your username. If there are any other contributors, add them by typing the first few letters of their name into the field. Their screen name should pop up. Select their name. To add more than one contributor other than yourself, click the "Add another item" button and repeat the same process.
+
+7. Next, in the "View Permissions and Groups", toggle the "Everyone can view this content" on or off to determine who can view the content.
+
+    If you choose on, complete the "Group association and view access" by adding groups by typing in its name into the field. 
+
+    If you choose off, add the people who have access to this memo by adding typing in their screen name and clicking it. Continue to completing the "Group association and view access" field just as instructed above.
```

Comment:
```suggestion
    If you choose to restrict access to the content, you can designate specific people or groups that will have view access. Add the people who should have view access to this memo by typing their screen name into the "People with view access" field. Add groups whose members should have view access to this memo by typing the group name into the "Group association and view access" field. **Note that if you give a group view access, it will also associate the content with that group. This means the content will appear on the group's page.**
```